README.rdoc

```diff
@@ -0,0 +1,28 @@
+== README
```

Comment:
Can get rid of this since there's already a README (or we can use this one as a starting point).